tools/pull-request-labeler/Azure.Sdk.Tools.PullRequestLabeler/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Lets avoid adding this if possible.

Reply:
Yep ... will remove. This whole PR may be able to be abandoned depending on where Alex goes with using office bot to process label assignments.